### ISSUE TITLE
New rule: Disallow mixes of different bitwise operators

### ISSUE THREAD

**Issue by feross** [feross/standard#801]
Proposal to make the [`no-mixed-operators`](http://eslint.org/docs/rules/no-mixed-operators) stricter so it includes bitwise operators.

This was suggested by @mafintosh in this comment: https://github.com/feross/standard/issues/566#issuecomment-278827089

**Comment by mafintosh** [comment]
tbc, my issue is when mixing bitwise ops with `+`, `*`, `===` etc. mixing them with other bitwise ops is pretty straightforward

**Comment by feross** [comment]
@mafintosh Yep, got it. The rule won't warn about operators that are equal precedence. Parens are only needed when mixing operators of different precedences.
